dictionary/Dictionary.js

```diff
@@ -45,6 +45,16 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the specified localization phrase exists in the
+	 * dictionary.
+	 *
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {boolean} {@code true} if the key exists and denotes a single
+	 * localization phrase.
```

Comment:
Please line up the "localization phrase" with "{boolean}" to match the documentation style of the rest of the file like this:
```
* @return {boolean} {@code true} if the...
*         localization phrase.
```